tests/torch/quantization/test_strip.py

```diff
@@ -255,10 +252,13 @@ def test_strip_quantization(mode, overflow_fix, num_bits):
 
     inference_model = compression_ctrl.strip()
     x_torch = inference_model(input_tensor)
-    # overflow_fix = False, because target_device = TRIAL
-    check_quantizer_operators(inference_model, 2**num_bits - 1, overflow_fix=False)
+    check_quantizer_operators(inference_model, 2**num_bits - 1)
+
+    assert torch.all(torch.isclose(x_nncf, x_torch, atol=1e-6)), f"{x_nncf.view(-1)} != {x_torch.view(-1)}"
 
-    assert torch.all(torch.isclose(x_nncf, x_torch)), f"{x_nncf.view(-1)} != {x_torch.view(-1)}"
```

Comment:
Where does the 1e-6, and absolute at that, comes from? We did not need to have any custom tolerances before, why do we need them now?